# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: third_party/libwebrtc/audio/channel_receive.h
# Commit: d0b311007c03
# Full Hash: d0b311007c033e83824f5f6996a70ab9e870f31f
# Author: Byron Campen [:bwc] <docfaraday@gmail.com>
# Date: 2021-11-01 18:17:33
# Regressor Bug: 1654112
# File Overlap Count: 5
# Description:
#   Bug 1654112 - Get RTCP BYE and RTP timeout handling working again (from Bug 1595479) r=mjf,dminor
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D106145
# ==============================================================================

diff -r daa9eac709dc -r d0b311007c03 third_party/libwebrtc/audio/channel_receive.h
--- a/third_party/libwebrtc/audio/channel_receive.h	Fri Feb 19 16:20:38 2021 +0000
+++ b/third_party/libwebrtc/audio/channel_receive.h	Fri Feb 19 15:56:41 2021 -0600
@@ -167,7 +167,8 @@
     absl::optional<AudioCodecPairId> codec_pair_id,
     rtc::scoped_refptr<FrameDecryptorInterface> frame_decryptor,
     const webrtc::CryptoOptions& crypto_options,
-    rtc::scoped_refptr<FrameTransformerInterface> frame_transformer);
+    rtc::scoped_refptr<FrameTransformerInterface> frame_transformer,
+    RtcpEventObserver* rtcp_event_observer);
 
 }  // namespace voe
 }  // namespace webrtc